.ci/setup_env_rhel.sh

```diff
@@ -52,8 +52,9 @@ done
 chronic sudo -E yum -y install $pkgs_to_install
 
 if [ "$(arch)" == "x86_64" ]; then
+	VERSION_ID="7"
 	echo "Install Kata Containers OBS repository"
-	obs_url="${KATA_OBS_REPO_BASE}/RHEL_7/home:katacontainers:releases:$(arch):master.repo"
+	obs_url="${KATA_OBS_REPO_BASE}/CentOS_${VERSION_ID}/home:katacontainers:releases:$(arch):master.repo"
```

Comment:
can you please add a comment above this line refering the packaging issue?

Reply:
@chavafg changes applied